tests/acceptance/actions-test.js

```diff
@@ -48,6 +54,20 @@ test('allows to chain actions', function(assert) {
   });
 });
 
+test('allows to chain aliased actions', function(assert) {
```

Comment:
Do you think `{ chainable: true }` would be a reasonable option to pass in, or is it better go with something more descriptive of what is actually happening (i.e., `{ returnAliasContext: true }`).